library/src/main/java/io/github/boguszpawlowski/composecalendar/Calendar.kt

```diff
@@ -75,11 +86,16 @@ public fun SelectableCalendar(
   horizontalSwipeEnabled: Boolean = true,
   calendarState: CalendarState<DynamicSelectionState> = rememberSelectableCalendarState(),
   dayContent: @Composable BoxScope.(DayState<DynamicSelectionState>) -> Unit = { DefaultDay(it) },
-  monthHeader: @Composable ColumnScope.(MonthState) -> Unit = { DefaultMonthHeader(it) },
-  weekHeader: @Composable BoxScope.(List<DayOfWeek>) -> Unit = { DefaultWeekHeader(it) },
+  monthHeader: @Composable ColumnScope.(CurrentState) -> Unit = { DefaultMonthHeader(it) },
+  weekHeader: @Composable ColumnScope.(CurrentState) -> Unit = { DefaultWeekHeader(it) },
+  weekDaysNames: @Composable BoxScope.(List<DayOfWeek>) -> Unit = { DefaultWeekDaysNames(it) },
   monthContainer: @Composable (content: @Composable (PaddingValues) -> Unit) -> Unit = { content ->
     Box { content(PaddingValues()) }
   },
+  weekContainer: @Composable (content: @Composable (PaddingValues) -> Unit) -> Unit = { content ->
+    Box { content(PaddingValues()) }
+  },
+  onSwipe: (LocalDate) -> Unit = { },
```

Comment:
Why is this necessary?

Reply:
When the month changes, I load data to display dots representing events.